Documentation/api.md

```diff
@@ -127,6 +127,7 @@ AlertmanagerSpec is a specification of the desired behavior of the Alertmanager
 | imagePullSecrets | An optional list of references to secrets in the same namespace to use for pulling prometheus and alertmanager images from registries see http://kubernetes.io/docs/user-guide/images#specifying-imagepullsecrets-on-a-pod | [][v1.LocalObjectReference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#localobjectreference-v1-core) | false |
 | secrets | Secrets is a list of Secrets in the same namespace as the Alertmanager object, which shall be mounted into the Alertmanager Pods. The Secrets are mounted into /etc/alertmanager/secrets/<secret-name>. | []string | false |
 | configMaps | ConfigMaps is a list of ConfigMaps in the same namespace as the Alertmanager object, which shall be mounted into the Alertmanager Pods. The ConfigMaps are mounted into /etc/alertmanager/configmaps/<configmap-name>. | []string | false |
+| configSecret | ConfigSecret is the name of a Kubernetes Secret in the same namespace as the AlertManager object, which contains configuration for this Alertmaanger instance. Defaults to 'alertmanager-<alertmanager-name>' The secret is mounted into /etc/alertmanager/config. | string | false |
```

Comment:
Two typos in “Alertmanager” (the m is not capitalized)

Reply:
Thanks, I've changed one instance of AlertManager in this comment to Alertmanager to be consistent with the rest of the docs in this section. I've also fixed an instance of 'Alertmaanger' to be spelled Alertmanager.